### DISCUSSION TITLE
UI to annotate a document with structured entities

### DISCUSSION THREAD

**Discussion by pudo** [alephdata/aleph#1526] [Ideas]
Have a split-screen UI where a user is shown a document (e.g. PDF, email, ...) and a form where they can create new entities (such as Ownership relations), events, companies, people which are then linked to the document (and page within) as a source.

**Comment by pudo** [comment]
@iaincollins we discussed this, I'm actually wondering if it may be an interesting starting point for the case mgmt.

**Comment by pudo** [comment]
Requires alephdata/aleph#457.

**Comment by jigsawsecurity** [comment]
Being able to make notes about a document would be a great feature.
